Tools/Scripts/hooks/prepare-commit-msg

```diff
@@ -316,11 +342,14 @@ def main(file_name=None, source=None, sha=None):
         return 0
 
     with open(file_name, 'w') as commit_message_file:
+        commit_title = os.environ.get('COMMIT_MESSAGE_TITLE')
         if sha:
             commit_message_file.write(subprocess.check_output(
                 ['git', 'log', sha, '-1', '--pretty=format:%B'],
                 **ENCODING_KWARGS
             ))
+        elif commit_title.startswith('Unreviewed, reverting'):
```

Comment:
This means that you won't trigger your revert function when someone runs a bare `git revert`. I think that's where we should start. If you rip out your `.git/hooks/prepare-commit-msg` hook and then run a `git revert`, you can see what a "raw" revert message is. I think we want to do something similar to what we do with `CHERRY_PICKING_RE` and `GIT_REFLOG_ACTION` action a few lines above this. I think I _tried_ to do something like that with `if source == 'merge' and not content.startswith('Revert'):`, but it doesn't look right, so you probably need to remove that line.

Reply:
Landed a fix at #20127 to set `git revert` message to its default. Future work will make that message closer in format to `git webkit revert`, but that will come after fixing `git webkit revert` first.